deploy/helm-chart/templates/deployment.yaml

```diff
@@ -52,6 +52,9 @@ spec:
         {{- else }}
         - --leader-election-disable
         {{- end }}
+        {{- if not .Values.nginxGateway.productTelemetry.enable }}
+        - --product-telemetry-disable
```

Comment:
question: looks like `--` reads the flag value, is that assumption correct?

Reply:
Yes, `--` is specified on the actual command line of the binary.